Source/WebCore/css/CSSWordBoundaryDetectionValue.cpp

```diff
@@ -40,14 +40,16 @@ CSSWordBoundaryDetectionValue::CSSWordBoundaryDetectionValue(WordBoundaryDetecti
 String CSSWordBoundaryDetectionValue::customCSSText() const
 {
     TextStream ts;
-    WTF::switchOn(value(), [&ts](WordBoundaryDetectionNormal) {
+    if (std::holds_alternative<WordBoundaryDetectionNormal>(value())) {
```

Comment:
Oh, I don't think we should do this. The benefit of `switchOn` is that it will cause a compiler error if not all the alternatives are listed.

Does this work instead:
```
WTF::switchOn(static_cast<std::variant<WordBoundaryDetectionNormal, WordBoundaryDetectionManual, WordBoundaryDetectionAuto>>(value()), ...
```

Reply:
(Could be even better with typedefs)

Reply:
Alternatively, can we do something like this?
```
class WordBoundaryDetection : public std::variant<WordBoundaryDetectionNormal, WordBoundaryDetectionManual, WordBoundaryDetectionAuto> {
public:
    ...
    using std::variant<WordBoundaryDetectionNormal, WordBoundaryDetectionManual, WordBoundaryDetectionAuto>::operator();
    ...
};
```

Reply:
Thank you for the suggestions. I tried out both. The second approach still resulted into a similar build error. The first approach worked, so I updated the patch using that solution.